docs/source/pre_application/cloud_node_management.rst

```diff
@@ -79,71 +28,64 @@ Now NuCypher Ops is installed we can create a droplet on Digital Ocean:
 
 .. code-block:: bash
 
-    nucypher-ops nodes create --network ibex --count 1 --cloudprovider digitalocean
+    nucypher-ops nodes create
 
 At this point you should see the droplet in your Digital Ocean dashboard.
 Now we can deploy the PRE Node:
 
 .. code-block:: bash
 
-    nucypher-ops ursula deploy --eth-provider $INFURA_RINKEBY_URL --nucypher-image nucypher/nucypher:experimental --payment-provider $INFURA_MUMBAI_URL --network ibex
+    nucypher-ops ursula deploy
+
+Follow the prompts to enter your ethereum and polygon provider URIs.
 
 This should produce a lot of log messages as the ansible playbooks install all the requirements and setup the node.
 The final output should be similar to:
 
 .. code-block:: bash
 
     some relevant info:
-    config file: "/SOME_PATH/nucypher-ops/configs/ibex/nucypher/ibex-nucypher.json"
-    inventory file: /SOME_PATH/nucypher-ops/configs/ibex-nucypher-2022-03-25.ansible_inventory.yml
+    config file: "/SOME_PATH/nucypher-ops/configs/mainnet/nucypher/mainnet-nucypher.json"
+    inventory file: /SOME_PATH/nucypher-ops/configs/mainnet-nucypher-2022-03-25.ansible_inventory.yml
     If you like, you can run the same playbook directly in ansible with the following:
-        ansible-playbook -i "/SOME_PATH/nucypher-ops/configs/ibex-nucypher-2022-03-25.ansible_inventory.yml" "src/playbooks/setup_remote_workers.yml"
+        ansible-playbook -i "/SOME_PATH/nucypher-ops/configs/mainnet-nucypher-2022-03-25.ansible_inventory.yml" "src/playbooks/setup_remote_workers.yml"
     You may wish to ssh into your running hosts:
         ssh root@123.456.789.xxx
     *** Local backups containing sensitive data may have been created. ***
-    Backup data can be found here: /SOME_PATH//nucypher-ops/configs/ibex/nucypher/remote_worker_backups/
+    Backup data can be found here: /SOME_PATH//nucypher-ops/configs/mainnet/nucypher/remote_worker_backups/
 
 This tells us the location of several config files and helpfully prints the IP address of our newly created node (you can also see this on the Digital Ocean dashboard).
 Let's ``ssh`` into it and look at the logs:
 
 .. code-block:: bash
 
     $ ssh root@123.456.789.xxx
-    root@nucypher-ibex-1:~#
-    root@nucypher-ibex-1:~# sudo docker logs --follow ursula
+    root@nucypher-mainnet-1:~#
+    root@nucypher-mainnet-1:~# sudo docker logs --follow ursula
     ...
     ! Operator 0x06E11400xxxxxxxxxxxxxxxxxxxxxxxxxxxx1Fc0 is not funded with ETH
```

Comment:
I get why the ordering of operations is done this way, but recognize that someone doing this who takes a long time to bond the operator or fund with ETH, will cause a ton of requests to infura for checking balance and bonding status. This is another argument for throttling Ursula's WorkTracker a bit better - it current runs in a tight loop between 1-2s (cc @vepkenez , @KPrasch )